lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,10 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Updating direction of scroll by calculating pixels.If the value is 0,
+		// the gradebook should be moved all the way to the left side of the screen.
+		// maxScroll is difference between table width and its parent width.
```

Comment:
You haven't actually addressed my comment. I still don't understand what you mean by "updating direction of scroll". By "scroll", do you mean moving the gradebook across the page? Normally, I think of "scroll" as the user changing the viewport, so they can view a different part of the page. It doesn't involve elements on the page moving.

I also looked up the [`.scrollLeft()`](http://api.jquery.com/scrollLeft/) function in jQuery's API, which is what the `updateHorizontalPosition()` function calls. It appears that `.scrollLeft()` does indeed change the viewport, rather than moving around elements on the page. As a result, you're not actually moving the gradebook all the way to the left side of the screen, so this comment is incorrect.

Please understand that **I don't know how this code is supposed to work.** I am trying to suggest comments for you to add to the code, and edits to those comments, but because I don't know how this code is supposed to works, my suggestions are likely to be wrong. I am relying on **you** to determine what this code is actually doing, and write comments that *actually improve understanding*, rather than you just taking my suggestions and dropping them into the code.